apps/shell/tash_security.c

```diff
@@ -77,6 +80,23 @@ void tash_check_security(int fd)
 			continue;
 		}
 
+		#ifdef CONFIG_TASH_REBOOT
+		if (!strncmp((unsigned char *)input, "reboot", 6)) {
```

Comment:
Is `6` correct? or `7` correct?

Reply:
We don't need to compare the end of the string(\0), so 6 is correct.

Reply:
But then, it will work with `rebootaa`. Is this ok?

Reply:
oh, I didn't think of that part... I'll fix it.